pkg/dunner/dunner.go

```diff
@@ -129,7 +134,8 @@ func process(configs *config.Configs, s *docker.Step, wg *sync.WaitGroup, args [
 	}
 }
 
-func passArgs(s *docker.Step, args *[]string) error {
+// PassArgs replaces argument variables,of the form '`$d`', where d is a number, with dth argument.
+func PassArgs(s *docker.Step, args *[]string) error {
```

Comment:
Do you want to expose these functions outside package? Its used only within the package.

Reply:
Yeah! If dunner is going to be used as a library, then the method that passes the arguments to the commands should be exposed.